### ISSUE TITLE
Dupe bug converting between redstone cable and freestanding redstone cable

### ISSUE THREAD

**Issue by CplPibald** [asiekierka/CharsetMC#269]
Using Charset 0.5.3.1 on MC 1.12.2 in SSP

**TL;DR: The Cable <--> Freestanding cable conversion recipes dupe when the input is a stack with more than one item.  Suggested fix at the bottom.**

Placing a stack of 3 redstone cable (RC) in a crafting grid shows output is 3 freestanding cable (FRC).  Clicking the output slot gives 3 FRC, but only uses one RC.  Clicking again gives two more, and again gives one more for a total of 3+2+1 = 6.  Shift-click the output slot to craft all items and see this dupe fast.  For a full input stack, this gives over 32 output stacks.

This bug is the same for all cable <--> freestanding recipes, including their reverse.

-----

The fix is in [RecipeWireConversion.java](https://github.com/CharsetMC/Charset/blob/2fb8ec4150f23624e32a4d9eac379e20bda24cdc/src/main/java/pl/asie/charset/lib/wires/RecipeWireConversion.java#L108) where the recipe output stack is set to a copy of the input stack, including stack size.  The expected output of this function is the conversion for one item, not for the stack.  

Adding the following line right before the return should fix the issue.

    newStack.setCount(1);

**Comment by asiekierka** [comment]
Fix finished, will be released in 0.5.4.1.
